### PR TITLE
Added Cloud SQL mysql with authorized network example

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**
Added example for inclusion on the following page:

https://cloud.google.com/sql/docs/mysql/authorize-networks

```release-note:none
```

### CODE REVIEW

**Review by rileykarson — APPROVED** [review]

File: mmv1/products/cgc/terraform.yaml [hunk 1]
```diff
@@ -51,6 +51,13 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           deletion_protection: "false"
         ignore_read_extra:
           - "deletion_protection"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_mysql_instance_authorized_network"
+        primary_resource_id: "instance"
+        vars:
+          mysql_instance_with_authorized_network: "mysql-instance-with-authoriz$
```
Comment (rileykarson) [inline mmv1/products/cgc/terraform.yaml:58]: Looks like there's a syntax error on this line!

**Review by rileykarson — CHANGES_REQUESTED** [review]
Oh wait, looks like we're failing to build

File: mmv1/products/cgc/terraform.yaml (hunk 1 above)
Comment (betsy-lichtenberg) [reply mmv1/products/cgc/terraform.yaml]: fixed

**Comment by betsy-lichtenberg** [comment]
Need to redo PR because of the master ---> main change.

**Pull request closed without merging**